interfaces/cython/cantera/cti2yaml.py

```diff
@@ -1585,16 +1586,18 @@ def convert(filename=None, output_name=None, text=None):
 
     try:
         if filename is not None:
-            with filename.open('r', encoding='latin-1') as f:
-                code = compile(f.read(), str(filename), 'exec')
+            with filename.open("r", encoding="utf-8") as f:
```

Comment:
I think this is deliberately `latin-1` to catch old Windows files. See here: https://stackoverflow.com/questions/13868140/showing-a-latin-1-character-in-a-utf-8-page for a description of the difference.

Reply:
The issue with `latin-1` is that having a regular `"` symbol in the header text causes crashes. The file `data/inputs/lithium_ion_battery.cti` is one example that causes failures.

Reply:
I'd rather handle cases like that with the `error` kwarg to `open`, setting it to either `ignore` or `replace`.  I think `latin-1` files are _extremely_ common and changing this behavior can cause unexpected effects.

Reply:
Sorry, I don't get what you mean?

Reply:
Oh, I meant to type *encoding*. Just
```
cti2yaml haca2.cti --encoding=utf-8
```
etc. (I *believe* `argparse` can handle that)

Reply:
Sure, sounds good. Perhaps `--input-encoding` would be more clear?

Reply:
Implemented `--input-encoding`.